content/en/logs/guide/best-practices-for-log-management.md

```diff
@@ -0,0 +1,206 @@
+7. Click **Save**.
+
+See [How to Set Up RBAC for Logs][9] for a step-by-step guide on how to set up and assign a role with specific permissions for an example use case.
+
+## Monitor log usage
+
+You can monitor your log usage, by setting up the following:
+
+- [Alerts for unexpected log traffic spikes](#alert-on-unexpected-log-traffic-spikes)
+- [Alert when an indexed log volume passes a specified threshold](#alert-when-an-indexed-log-volume-passes-a-specified-threshold)
+
+### Alert on unexpected log traffic spikes
+
+#### Log usage metrics
+
+By default, [log usage metrics][10] are available to track the number of ingested logs, ingested bytes, and indexed logs. These metrics are free and kept for 15 months:
+
+- `datadog.estimated_usage.logs.ingested_bytes`
+- `datadog.estimated_usage.logs.ingested_events`
+
+See [Anomaly detection monitors][11] for steps on how to create anomaly monitors with the usage metrics.
+
+**Note**: Datadog recommends setting the unit to `byte` for the `datadog.estimated_usage.logs.ingested_bytes` in the [metric summary page][12]:
+
+{{< img src="logs/guide/logs_estimated_bytes_unit.png" alt="The metric summary page showing the datadog.estimated_usage.logs.ingested_bytes side panel with the unit set to byte" style="width:70%;">}}
+
+#### Anomaly detection monitors
+
+To create an anomaly detection monitor to alert on any unexpected log indexing spikes:
+
+1. Navigate to [Monitor > New Monitor][13] and select **Anomaly**.
+2. In the **Define the metric** section, select the `datadog.estimated_usage.logs.ingested_events` metric.
+3. In the **from** field, add the `datadog_is_excluded:false` tag to monitor indexed logs and not ingested ones.
+4. In the **sum by** field, add the `service` and `datadog_index` tags, so that you are notified if a specific service spikes or stops sending logs in any index.
+5. Set the alert conditions to match your use case. For example, set the monitor to alert if the evaluated values are outside of an expected range.
+6. Add a title for the notification and a message with actionable instructions. For example, this is a notification with contextual links:
+    ```text
+    An unexpected amount of logs has been indexed in the index: {{datadog_index.name}}
+
+    1. [Check Log patterns for this service](https://app.datadoghq.com/logs/patterns?from_ts=1582549794112&live=true&to_ts=1582550694112&query=service%3A{{service.name}})
+    2. [Add an exclusion filter on the noisy pattern](https://app.datadoghq.com/logs/pipelines/indexes)
+    ``` 
+7. Click **Create**.
+
+### Alert when an indexed log volume passes a specified threshold
+
+Set up a monitor to alert if an indexed log volume in any scope of your infrastructure (for example, `service`, `availability-zone`, and so forth) is growing unexpectedly.
+
+1. Navigate to the [Log Explorer][14].
+2. Enter a [search query][15] that captures the log volume you want to monitor. Keep the query empty to monitor all the logs from that index.
```

Comment:
I think you need to add the index to this query, otherwise the search query is pulling all log volumes isn't it? `index:main`

Reply:
Thanks! Added the info.